File: android/util/VelocityTracker_2017_4_14/app/src/main/java/top/goluck/velocitytracker_2017_4_14/MainActivity.java
package top.goluck.velocitytracker_2017_4_14;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends SwipeBackActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.text).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                start();
            }
        });
        start();
    }

    public void start(){
        Intent m = new Intent(MainActivity.this,Main2Activity.class);
        startActivity(m);
    }
}

File: android/kotlin/Kotlin_2017_8_6/app/src/main/java/top/goluck/kotlin_2017_8_6/MainActivity.kt
package top.goluck.kotlin_2017_8_6

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setText()
        startActivity()
        txt.setOnClickListener { startActivity() }
    }

    fun setText() {
        txt.text = "全局静态Long类型变量："+OneActivity.TIMER_DELAY + "\n" + getClassName<MainActivity>(this) +"\n"+ OneActivity.mUser.name
    }

    fun startActivity(){
        var mInt = Intent(this, OneActivity::class.java)
        startActivity(mInt)
    }
}